### PR TITLE
QWinFF: new recipe

### PR DESCRIPTION
**Pull request by miqlas**
Till MediaConverter matures

### CODE REVIEW

File: media-video/qwinff/qwinff-0.2.1.recipe [hunk 1]
```diff
@@ -0,0 +1,71 @@
...
+BUILD()
+{
+	make $jobArgs PREFIX=$appsDir/qwinff
+}
+
+INSTALL()
+{
+	make PREFIX=$appsDir/qwinff install
+
+	mv $appsDir/qwinff/bin/* $appsDir/qwinff
```
Comment (diversys) [inline media-video/qwinff/qwinff-0.2.1.recipe]: $appsDir/QWinFF

Comment (miqlas) [reply media-video/qwinff/qwinff-0.2.1.recipe]: Fixed.

Comment (diversys) [reply media-video/qwinff/qwinff-0.2.1.recipe]: Forgot to push your fixes?

Comment (miqlas) [reply media-video/qwinff/qwinff-0.2.1.recipe]: yeah, it seems.

**Comment by miqlas** [comment]
Mission completed!

**Comment by waddlesplash** [comment]
committed directly.

**Pull request closed without merging**